app/stac_api/models.py

```diff
@@ -301,6 +301,15 @@ class Meta:
         blank=False, null=False, editable=False, max_length=56, default=compute_etag
     )
 
+    update_interval = models.IntegerField(
+        default=-1,
+        null=False,
+        blank=False,
+        validators=[MinValueValidator(-1)],
+        help_text="Minimal update interval in seconds "
+        "in which the underlying assets data are updated."
```

Comment:
should it be "underlying items" here? (Even if the value of the item's is that of the asset's, and hence it will be the same here.)

Reply:
Nope the `update_interval` is always for the asset data/file update interval. On item/collection it is the minimal value of all their assets but still refer to the asset data update interval.